tools/install.sh

```diff
@@ -9,6 +9,12 @@
 
 set -euo pipefail
 
+# Script must be run from tools directory.
+if [[ "$PWD" != *'/tools'* ]]; then
+    echo "Please run this script from the tools directory."
+    exit 2
```

Comment:
```suggestion
    exit 1
```

The error codes have a specific meaning for bash and I don't think 2 matches what's happening here.